src/Translator/DummyTranslator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Validator\Translator;
+
+use Laminas\I18n\Translator\TranslatorInterface as I18nTranslatorInterface;
+
+class DummyTranslator implements I18nTranslatorInterface
+{
+    /** @inheritDoc */
+    public function translate($message, $textDomain = 'default', $locale = null)
+    {
+        return $message;
+    }
+
+    /** @inheritDoc */
+    public function translatePlural($singular, $plural, $number, $textDomain = 'default', $locale = null)
+    {
+        // phpcs:disable SlevomatCodingStandard.Operators.DisallowEqualOperators
+        return $number == 1 ? $singular : $plural;
```

Comment:
Why not type casting?
```suggestion
        return (int) $number === 1 ? $singular : $plural;
```

Reply:
If I add the typecasting, psalm shows a complaint `RedundantCastGivenDocblockType - src/Translator/DummyTranslator.php:20:16 - Redundant cast to int given docblock-provided type (see https://psalm.dev/263)`

Reply:
Should be added to `psalm-baseline.xml`. Example can be found in other repositories:

- https://github.com/laminas/laminas-paginator/blob/f9cd4cbc5602e5335cc332f9733d433a94895098/psalm-baseline.xml#L139
- https://github.com/laminas/laminas-paginator/blob/f9cd4cbc5602e5335cc332f9733d433a94895098/psalm-baseline.xml#L206-L214

Reply:
done